HiP-DataStore/Controllers/ExhibitPagesController.cs

```diff
@@ -280,6 +283,192 @@ public IActionResult GetReferenceInfo(int id)
             return ReferenceInfoHelper.GetReferenceInfo(ResourceTypes.ExhibitPage, id, _entityIndex, _referencesIndex);
         }
 
+        /// <summary>
+        /// Returns the review to the exhibit page with the given ID
+        /// </summary>
+        /// <param name="id">ID of the exhibit page the review belongs to</param>
+        [HttpGet("Pages/Review/{id}")]
+        [ProducesResponseType(typeof(ReviewResult), 200)]
+        [ProducesResponseType(400)]
+        [ProducesResponseType(403)]
+        [ProducesResponseType(404)]
+        public IActionResult GetRreview(int id)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            if (!_entityIndex.Exists(ResourceTypes.ExhibitPage, id))
+                return NotFound(ErrorMessages.ContentNotFound(ResourceTypes.ExhibitPage, id));
+
+            if (!_reviewIndex.Exists(ResourceTypes.ExhibitPage.Name, id))
+                return NotFound(ErrorMessages.ReviewNotFound(ResourceTypes.ExhibitPage, id));
+
+            var reviewId = _reviewIndex.GetReviewId(ResourceTypes.ExhibitPage.Name, id);
+            var review = _db.Get<Review>((ResourceTypes.Review, reviewId));
```

Comment:
You should get the object from the event stream (by using the extension method on the event stream) instead of the database